packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -66,16 +66,16 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
   }
 
   public render() {
-    let { className } = this.props;
+    let { className, navigationLandmarkAriaLabel } = this.props;
```

Comment:
@cschleiden the aria-label does need to go on the focus zone because it's the label for the div that has role="navigation" which is the focus zone. we can definitely change the name to ariaLabel. and i don't know if it is supposed to encapsulate the overflow, that overflow button is not in any ARIA docs, let me try and find some guidance